### ISSUE TITLE
Register the clusters ACM Application manifests are located ...

### ISSUE THREAD

**Issue by adelton** [opendatahub-io/ai-edge#81]
The README.md https://github.com/opendatahub-io/ai-edge#infrastructure-configuration step 4 starts with sentence
> 4. Register the clusters [ACM Application](https://access.redhat.com/documentation/en-us/red_hat_advanced_cluster_management_for_kubernetes/2.8/html/applications/managing-applications) manifests are located in [acm/registration](https://github.com/opendatahub-io/ai-edge/blob/main/acm/registration) to register and configure the target environments required for the AI at the Edge use cases.

It's not clear if it should be

"Register the clusters. ACM Application manifests ..."

or

"Register the clusters' ACM Application. Manifests ..."

Please reword it to make it clear what the user should do.

**Comment by adelton** [comment]
OK. Is it Register the cluster a.k.a. the Import cluster functionality of ACM, or something else? Are the ACM Application manifests used to register the clusters, or do you register the clusters and then use the ACM Application manifests for some further steps?

**Comment by LaVLaS** [comment]
Due to the missing indentation on the second line. the markdown formatting is joining lines together

**Comment by adelton** [comment]
I filed https://github.com/opendatahub-io/ai-edge/pull/83 to add the newline and indentation.

But I still fail to understand if a full cluster registration (for example Import cluster per https://access.redhat.com/documentation/en-us/red_hat_advanced_cluster_management_for_kubernetes/2.8/html-single/clusters/index#importing-managed-cluster-console) is needed, or if the https://access.redhat.com/documentation/en-us/red_hat_advanced_cluster_management_for_kubernetes/2.8/html/applications/managing-applications takes a fine grained approach of only making some resources available across cluster boundaries (in which case the "Register the clusters" is wrong).
